<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-4VBBEJ4" ignored="2" name="Default test(failed)" tests="2" failures="2" timestamp="2022-10-20T11:13:08 IST" time="47.759" errors="0">
  <testcase name="tc2" time="4.192" classname="org.login.Testing">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.login.Testing.tc2(Testing.java:46)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- tc2 -->
  <testcase name="tc1" time="43.528" classname="org.login.Testing">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.login.Testing.tc1(Testing.java:37)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- tc1 -->
  <testcase name="tc0" time="0.0" classname="org.login.Testing">
    <ignored/>
  </testcase> <!-- tc0 -->
  <testcase name="tc3" time="0.0" classname="org.login.Testing">
    <ignored/>
  </testcase> <!-- tc3 -->
</testsuite> <!-- Default test(failed) -->
